src/node/miner.h

```diff
@@ -30,7 +30,7 @@ class ChainstateManager;
 namespace Consensus { struct Params; };
 
 namespace node {
-static const bool DEFAULT_PRINTPRIORITY = false;
+static const bool DEFAULT_PRINT_MODIFIED_FEE = false;
```

Comment:
nano nit: would call it "DEFAULT_PRINT_FEERATE" as the "modified" word is internal and might change over time.

Reply:
I didn't change the external option, just the internal name - as requested here https://github.com/bitcoin/bitcoin/pull/30324#discussion_r1652212085